src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -121,16 +135,19 @@ export function useSwapInputsController({
   });
 
   const formattedInputNativeValue = useDerivedValue(() => {
+    if (inputMethod.value === 'inputNativeValue') {
+      return addSymbolToNativeDisplayWorklet(inputValues.value.inputNativeValue, currentCurrency);
+    }
     if (
       (inputMethod.value === 'slider' && percentageToSwap.value === 0) ||
       !inputValues.value.inputNativeValue ||
       !isNumberStringWorklet(inputValues.value.inputNativeValue.toString()) ||
       equalWorklet(inputValues.value.inputNativeValue, 0)
     ) {
-      return convertAmountToNativeDisplayWorklet(0, currentCurrency);
+      return convertAmountToNativeDisplayWorklet(0, currentCurrency, false, true);
```

Comment:
I didn't want to convert the `convertAmountToNativeDisplayWorklet` function to accept object params yet, but just updating to add a final parameter to ignore currency alignment for swaps inputs (e.g, if a native currency is right-aligned, display it still with the symbol on the left)